java-native/src/main/java/com/baeldung/unsatisfiedlink/JniUnsatisfiedLink.java

```diff
@@ -0,0 +1,15 @@
+package com.baeldung.unsatisfiedlink;
+
+public class JniUnsatisfiedLink {
+
+    public static final String LIB_NAME = "test";
+
+    public static void main(String[] args) {
+        System.loadLibrary(LIB_NAME);
+        new JniUnsatisfiedLink().test();
+    }
+
+    public native String test();
+
+    public native String inexistentDllMethod();
```

Comment:
Although inexistent is a valid adjective and very natural for languages of Latin origin in English is more natural the use of nonexistent.